### PR TITLE
Improved getting started docs

### PR DESCRIPTION
**Pull request by ferama**
Better explaination of how to send messages to a channel using websockets.
Added a basic nodejs example

### CODE REVIEW

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by anovakovic01 — COMMENTED** [review]
@ferama You will have to sign off you commit using `git commit -s`.

File: docs/getting-started.md
Comment (drasko) [inline docs/getting-started.md]: Move URL in another `var`, and move API key in yet another `var`. This will make it easier for user to replace key and URL if needed.

**Review by drasko — CHANGES_REQUESTED** [review]

**Comment by drasko** [comment]
@ferama thanks for sending PR. I proposed small modification to make your example more generic.

It is good enough, if you do not want to bother we can merge as-is.

@anovakovic01 @nmarcetic opinions?

File: docs/getting-started.md
Comment (ferama) [reply docs/getting-started.md]: Is just a quick example, not real code. So keeping it small and simple could be a good thing I think

Comment (anovakovic01) [reply docs/getting-started.md]: I agree with @ferama here.

Comment (drasko) [reply docs/getting-started.md]: OK, merging then.

**Review by drasko — DISMISSED** [review]
LGTM

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

**Comment by chombium** [comment]
@drasko @ferama I think that few comments would clear up the things:

```javascript
// do not verify self-signed certificates if you are using one
process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"

// cbf02d60-72f2-4180-9f82-2c957db929d1  is an example of a thing_auth_key
const ws = new WebSocket('wss://localhost/ws/channels/1/messages?authorization=cbf02d60-72f2-4180-9f82-2c957db929d1')
```

But of course that's not a must and a little bit of googling can also clear up the things :)

**Review by anovakovic01 — COMMENTED** [review]
Please use `git rebase` and sign off all your commits. We cannot merge it before you do that.

**Comment by drasko** [comment]
@ferama @chombium agreed on this comments - please add some comments and let's merge ;)

**Review by dborovcanin — DISMISSED** [review]

**Comment by drasko** [comment]
@ferama I can merge this, but it would be good if you can add this comment that @chombium proposed.

**Comment by ferama** [comment]
Sorry guys, I was very busy. I pushed the changes

**Review by anovakovic01 — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Comment by drasko** [comment]
Merged, thanks @ferama!

**Pull request merged**
